tldw_Server_API/app/api/v1/router_groups/content.py

```
@@ -1,14 +1,986 @@
+            kanban_checklists_router,
+            kanban_comments_router,
+            kanban_search_router,
+            kanban_links_router,
+            kanban_workflow_router,
+        ):
+            specs.append(RouterSpec(
+                router=router,
+                prefix=f"{API_V1_PREFIX}/kanban",
+                tags=("kanban",),
+                route_key="kanban",
+            ))
```

Ruff (0.15.11)

[warning] Use `list.extend` to create a transformed list

(PERF401)